spec/index.html

```diff
@@ -362,6 +350,16 @@ <h3>Version Announcement</h3>
       -->
     </pre>
 
+    <p class="note">When using RDF 1.2-specific features, such as
+      <a data-cite="RDF12-CONCEPTS#dfn-base-direction">initial text direction</a>,
+      the specific RDF version should be announced using
+      either the <a href="#grammar-production-sparqlVersion"><code>VERSION</code></a>
+      or <a href="#grammar-production-version"><code>@version</code></a>
+      directives early in the document.
+      This allows parsers that do not support these features to detect
+      the presense of such features early.
```

Comment:
```suggestion
      the presense of such features early, and therefore 
      cease processing a job that would ultimately fail.
```

Reply:
I'm not that happy with the new proposed wording. Perhaps: "and potentially halt processing early."

Reply:
Maybe?
```suggestion
      the presense of such features early, and potentially 
      inform the user, giving them an opportunity to stop the job
      or otherwise act on the fact that some amount of the input data 
      will not be processed as desired.
```